src/modules/control_allocator/module.yaml

```diff
@@ -90,14 +89,14 @@ parameters:
             description:
                 short: Servo ${i} slew rate limit
                 long: |
-                  Minimum time allowed for the servo input signal to pass through
-                  the full output range. A value x means that the servo signal
-                  can only go from -1 to 1 in minimum x seconds.
+                  Forces the servo output signal to take at least the configured time (in seconds)
+                  to traverse its full range [-100%, 100%].
```

Comment:
Is the range of a servo defined from -1 to 1? I.e. can a servo be defined to have a range of 0 - 100%? If it can, I'd be tempted to remove the `[-100%, 100%]`

Reply:
From the actuator doc example "They also have a slew rate of 0.5/s, meaning that it will take 2s to fully deflect them (a slew rate on the flaps is generally recommended to reduce the disturbances their movement creates)."

So I think this is from 0 to 1.

Reply:
I actually don't know and would need to check in code. Likely @sfuhrer knows by heart and in case it's not super clear certainly how it should be.

Reply:
The servo movement is always defined from -1 to 1. A flap is at -1 if at neutral and at 1 if fully deflected. 
Does that answer your question?